cpp/cmake/thirdparty/get_cuco.cmake

```diff
@@ -16,12 +16,8 @@
 
 function(find_and_configure_cuco VERSION)
 
-    if(TARGET cuco::cuco)
-      return()
-    endif()
```

Comment:
This was added by #1746 since we were seeing CPM errors about the alias target `cuco::cuco` already existing, since I believe raft also defines this.  Is this now safe to remove if we use raft?  Better question: if the target is already defined, will these changes result in an error?  If not, is there still an advantage in returning early if this was already defined?

Reply:
It is now safe to remove these guards when the target ( `cuco::cuco` ) is part of the `GLOBAL_TARGETS` list of `rapids_cpm_find`

This is due to an improvement to rapids-cmake ( https://github.com/rapidsai/rapids-cmake/pull/51 ) where it now rolls these checks into `rapids_cpm_find`.